tasks/package.js

```diff
@@ -50,6 +50,15 @@ module.exports = function(grunt) {
     tasks.push('clean:packages');
     tasks.push('copy:package');
 
+    // Rebuild composer dependencies without dev.
+    grunt.config(['composer'], {
+      options: {
+        flags: ['no-dev'],
+        cwd: destPath
+      }
+    });
+    tasks.push('composer:install');
```

Comment:
@grayside That would only be a problem if also executed in the packaged output, right? The dev dependencies will still exist in the project root.